uobtheatre/productions/migrations/0027_alter_production_options.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.16 on 2023-02-24 19:11
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('productions', '0026_auto_20221210_2056'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
```

Comment:
So I was thinking it would be a good idea to give existing production teams this new permission. We could either do this manually once deployed or we can do it the proper way, with a mutation in this migration so that after it creates the permission, it gets all the users with explicitly assigned "change_production" permissions and grants them the new permission. What do you think?

An example of where we did this previously; when we added the check in timestamp to bookings: https://github.com/BristolSTA/uobtheatre-api/blob/dev/uobtheatre/bookings/migrations/0006_alter_ticket_checkin_to_datetime.py

Reply:
But we don't want everybody with "change_production" permission to have the new permission (like SM's and the approval team). would it not be easier to do it manually or could we find a way to make a distinction?

Reply:
We'd only apply it to people who have "change_production" applied explicitly to the object (this equates to the production team at the moment, because other people have the global permission through groups instead). Essentially, we would be grandfathering in the fact that people who currently have change_production applied to a specific production should be able to create a comp booking. I don't mind doing it manually, but need to make sure we remember to do this when it goes live